pages/linux/nsenter.md

```diff
@@ -0,0 +1,16 @@
+# nsenter
+
+> Run a new command in a running process' namespace.
+> Particularly useful for docker images of chroot jails.
+
+- Run command in existing processes network namespace:
+
+`nsenter -t {{pid}} -n {{command}} {{command_arguments}}`
+
+- Run command in existing processes process namespace:
```

Comment:
This sentence is kind of confusing. Can we find a way of rewriting it?

Reply:
@schneiderl: Agreed. I was excited because I finally found a use-case for it.

How about `Run a new command in an existing processes ps-table namespace`?

Reply:
That is way better in my opinion. I say we go for it.